packages/charts/src/core/common/types.ts

```diff
@@ -185,11 +185,12 @@ export interface IChartSeries<A extends IAccessors> extends IDataSeries<A> {
      * uses 'x' and 'y' as the keys for its scales.
      */
     scales: Scales;
+
+    renderState?: RenderState;
 }
 
 export interface IChartAssistSeries<A extends IAccessors> extends IChartSeries<A> {
     showInLegend?: boolean;
-    renderState?: RenderState;
     preprocess?: boolean;
```

Comment:
May as well add jsdoc comments for this interface as well (if you're up for it).